roles/aws.ec2-autoscaling-group/tasks/main.yml

```diff
@@ -8,6 +8,7 @@
       - /usr/bin/easy_install pip
       - /usr/local/bin/pip install --upgrade boto3 boto awscli requests psutil {{ asg_additional_python_pip_packages }}
       {{ asg_additional_user_data_bootcmd }}
+      {{ _efs_additional_bootcmd }}
```

Comment:
efs is not a core module, therefore core modules shouldn't be changed. Instead, leverage the already available  `asg_additional_user_data_bootcmd` in your playbook